app/src/main/java/com/kickstarter/ui/fragments/PledgeFragment.kt

```diff
@@ -443,17 +463,24 @@ class PledgeFragment : BaseFragment<PledgeFragmentViewModel.ViewModel>(), Reward
         pledge_details.y = pledge_root.height.toFloat()
     }
 
-    private fun updatePledgeCardSelection(positionAndSelected: Pair<Int, Boolean>) {
+    private fun updatePledgeCardSelection(positionAndSelected: Pair<Int, CardState>) {
```

Comment:
Can you rename this to `positionAndCardState` according to the implementation `CardState` is more appropriate.